crates/jstz_node/src/services/logs/mod.rs

```diff
@@ -122,42 +128,29 @@ mod persistent_logging {
                 db.logs_by_address_and_request_id(addr, request_id).await
             }
         }
-        .map_err(Error::InternalError)
     }
 }
 #[cfg(feature = "persistent-logging")]
 use persistent_logging::*;
 
-#[get("{address}/stream")]
-async fn stream_logs(
-    broadcaster: Data<Broadcaster>,
-    path: Path<String>,
-) -> std::io::Result<impl Responder> {
-    let address = path.into_inner();
-
-    // TODO: add better error
-    let address = Address::from_base58(&address)
-        .map_err(|e| std::io::Error::new(InvalidInput, e))?;
-
-    Ok(broadcaster.new_client(address).await)
-}
+use super::error::{ServiceError, ServiceResult};
 
 pub struct LogsService;
 
-impl Service for LogsService {
-    fn configure(cfg: &mut ServiceConfig) {
-        let scope = Scope::new("/logs").service(stream_logs);
+impl LogsService {}
 
-        #[cfg(not(feature = "persistent-logging"))]
-        cfg.service(scope);
+impl Service for LogsService {
+    fn router() -> Router<AppState> {
+        let routes = Router::new().route("/{address}/stream", get(stream_log));
```

Comment:
Does `/{...}` also work with axum? They only mention `/:` in the [doc](https://docs.rs/axum/latest/axum/struct.Router.html#captures).

Reply:
Oops, missed it. Updated.